src/app/pages/main/main.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { PureComponent } from 'react';
 import { InjectedIntlProps } from 'react-intl';
-import { v1 } from 'uuid';
+// import { v1 } from 'uuid';
```

Comment:
why is this code commented out?

Reply:
.map function which uses this import causes problems - to further investigation

Reply:
it works when you run the app with json-server enabled, by `yarn start:mocked` or `npm run start:mocked`